### ISSUE TITLE
Live Trading demo in Getting Started Guide does not work because Key Reference to exchange key is missing

### ISSUE THREAD

**Issue by DanielHeckrath** [Superalgos/Superalgos#1135]
TLDR: Key Reference to the exchange key is missing on live trading task. Live trading works after key reference is created. Since I have fixed the issue on my system, I might be able to provide a PR, but right now I have no Idea what the exact process is to fix issues in a workspace.

## Context

**Operating System:** Windows 10

**Software version:** today's master branch

**Workspace version:** default Workspace.json (Getting Started Guide)

**Reproducible:** the issue may be reproduced.

### 1. Explain what the problem seems to be.

When getting to the live trading step in the getting started guide, an error is logged in the terminal. The problem here is, that there is no key reference present on the live trading task, that references an exchange key.

### 2. Clearly explain what you wish to accomplish.

As a user I want to be able to run the Getting Started Guide from start to finish, including the live trading demo

### 3. Explain, step by step, what you've done to achieve the goal. The Team will follow these steps to reproduce the issue, so be precise.

- Clone superalgos
- Run `node setup`
- Run `node run`
- Proceed in getting started guide up to the live trading step
- Run live trading task
- Run live trading session

### 4. Explain what is the result you expect from your actions.

Expectation here is that the live trading session reports that it is running.
I would either expect here, that the key reference is already setup referencing the exchange key in the crypto ecosystem, or that at that point creation of references is mentioned in the tutorial.

### 5. Explain what happens instead of the expected results.

No error in the UI and the user does not get any indication that the live trading session is running.

### 6. Explain what errors you have found on-screen, on the browser console application, or in the ```Log-Files``` folder.

Log message in the superalgos client terminal reporting that the key reference is missing

*********** ['2021-06-12T18:23:43.941Z',2,'Trading Session','[ERROR] initialize -> checkKey -> Key Reference Node Missing'] @ Masters-Low-Frequency-Trading

**Comment by Luis-Fernando-Molina** [comment]
Thanks for reporting this, it should be fixed now. You will need to load the Getting Started Plugin Again and set up your exchange keys again since that will replace your working copy.
